src/containers/UserSettings/settings.tsx

```diff
@@ -167,6 +174,7 @@ export const experimentsSection: SettingsSection = {
     settings: [
         useNodesEndpointSetting,
         usePaginatedTables,
+        useHideDomainDatabase,
```

Comment:
Should it be in experiments? It's not actual experiment. Maybe introduce some section with advanced settings, that help you to customise UI (no idea, how it could be called)?

Reply:
moved to general